CefSharp.OffScreen.Example/Program.cs

```diff
@@ -28,20 +29,76 @@ public static void Main(string[] args)
             // You need to replace this with your own call to Cef.Initialize();
             CefExample.Init();
 
+            var cachepath1 = Path.Combine(Path.Combine(Path.GetTempPath(), "Reporting"), Guid.NewGuid().ToString("n"));
+            var cachepath2 = Path.Combine(Path.Combine(Path.GetTempPath(), "Reporting"), Guid.NewGuid().ToString("n"));
+
+            Directory.CreateDirectory(cachepath1);
+            Directory.CreateDirectory(cachepath2);
+
+            var cookieManager = new CookieManager(cachepath1, false);
+
+            var requestContext = new RequestContext(cookieManager);
+
+            var settings = new BrowserSettings()
+            {
+                RequestContext = requestContext
+            };
+
+            var cookieManager2 = new CookieManager(cachepath2, false);
+
+            var requestContext2 = new RequestContext(cookieManager2);
+
+            var settings2 = new BrowserSettings()
+            {
+                RequestContext = requestContext2
+            };
+
             // Create the offscreen Chromium browser.
-            using (browser = new ChromiumWebBrowser(testUrl))
+            //using (browser = new ChromiumWebBrowser(testUrl, new BrowserSettings() }*/))
+            using (browser = new ChromiumWebBrowser(null, settings ))
+            using (browser2 = new ChromiumWebBrowser(null, settings2))
             {
+                browser.BrowserInitialized += (sender, argsi) =>
+                {
+                    browser.Load(testUrl);
+                };
+
+                browser2.BrowserInitialized += (sender, argsi) =>
+                {
+                    browser2.Load(testUrl);
+                };
+
 
                 // An event that is fired when the first page is finished loading.
                 // This returns to us from another thread.
                 if (captureFirstRenderedImage)
                 {
                     browser.ResourceHandlerFactory.RegisterHandler(testUrl, ResourceHandler.FromString("<html><body><h1>CefSharp OffScreen</h1></body></html>"));
-                    browser.ScreenshotAsync().ContinueWith(DisplayBitmap);
+                    browser.ScreenshotAsync().ContinueWith(t => DisplayBitmap(t, "CefSharp screenshot Zoomlevel 0.png"));
                 }
                 else
                 {
                     browser.FrameLoadEnd += BrowserFrameLoadEnd;
+                    browser.FrameLoadStart += (s, argsi) =>
+                    {
+                        if (argsi.IsMainFrame)
```

Comment:
Braces around if blocks even if single line - there's more instances of this also.